sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -2149,6 +2149,7 @@ pi_result cuda_piextQueueGetNativeHandle(pi_queue queue,
 ///
 /// \return TBD
 pi_result cuda_piextQueueCreateWithNativeHandle(pi_native_handle, pi_context,
+                                                bool ownNativeHandle,
```

Comment:
you should probably add a fake use like `(void)ownNativeHandle'`

Reply:
done